Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -521,7 +521,7 @@ extension ARExample {
     /// - Returns: The new scene.
     private func imageryScene() -> AGSScene {
         // Create scene with the streets basemap.
-        let scene = AGSScene(basemapType: .imageryWithLabels)
+        let scene = AGSScene(basemapStyle: .arcGISImagery)
```

Comment:
Another oddity here: `.arcGISImageryLabels` will only render the streets. `.arcGISImagery` renders okay for me.

Reply:
Can you reproduce it with a simple SceneView?  If you can, check the current issues in the cocoa repo to see if there's already an issue for it.

Reply:
Yes, I've actually been able to reproduce this now on the Swift SDK as well in both a scene and map view.

I've opened a discussion in Slack [here](https://esri-runtime.slack.com/archives/C0135F8SW8P/p1650319345892809).

Reply:
Resolving - this is expected behavior